src/System.Device.Gpio/System/Device/Gpio/Drivers/UnixDriver.Windows.cs

```diff
@@ -14,6 +14,11 @@ public UnixDriver()
             throw new PlatformNotSupportedException();
         }
 
+        public static UnixDriver Create()
+        {
+            return new UnixDriver();

```

Comment:
>UnixDriver [](start = 23, length = 10)

is this intended to throw?

Reply:
yes, it will eventually since it will call the constructor which will throw. Are you saying I should simply throw directly instead?